public/app/plugins/panel/pmm-update/components/ConfirmUpdateModal/ConfirmUpdateModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { Button, Modal, useStyles2 } from '@grafana/ui';
+
+import { useVersionDetails } from '../../hooks';
+import { ConfirmUpdateModalProps } from '../../types';
+
+import { Messages } from './ConfirmUpdateModal.messages';
+import { getStyles } from './ConfirmUpdateModal.styles';
+
+export const ConfirmUpdateModal: React.FC<ConfirmUpdateModalProps> = ({ isOpen, onCancel, onConfirm }) => {
```

Comment:
can we instead do a direct import for `FC`?

Reply:
Fixed.